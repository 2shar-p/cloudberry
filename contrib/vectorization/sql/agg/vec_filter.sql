SET vector.enable_vectorization = on;
SET optimizer=on;

explain (verbose on, costs off) select c2 from trow where c2 > 15 order by c2;
select c2 from trow where c2 > 15 order by c2;
