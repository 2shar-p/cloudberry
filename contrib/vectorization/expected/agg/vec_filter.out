SET vector.enable_vectorization = on;
SET
SET optimizer=on;
SET

explain (verbose on, costs off) select c2 from trow where c2 > 15 order by c2;
 QUERY PLAN
-------------------------------------------------
 Gather VecMotion 2:1  (slice1; segments: 2)
   Output: c2
   Merge Key: c2
   ->  Vec Sort
         Output: c2
         Sort Key: trow.c2
         ->  Vec Seq Scan on public.trow
               Output: c2
               Filter: (trow.c2 > 15)
 Optimizer: Pivotal Optimizer (GPORCA)
 Settings: vector.enable_vectorization=on, optimizer=on
(11 rows)
select c2 from trow where c2 > 15 order by c2;
 c2
----
 16
 16
 17
 17
 18
 18
 19
 19
 20
 20
(10 rows)
